extensions/terminal-suggest/ThirdPartyNotices.txt

```diff
@@ -0,0 +1,32 @@
+THIRD-PARTY SOFTWARE NOTICES AND INFORMATION
```

Comment:
@rzhao271 do you know if we need a `cgmanifest.json` here as well? We inlined some files from an OSS project

Reply:
I'm not a lawyer. My pattern-matching suggests we could use one, ref https://github.com/microsoft/vscode/blob/13bfb57923faa5ea0ffbf5878fcdeb64cd63d942/src/vs/base/browser/dompurify/cgmanifest.json.